### ISSUE TITLE
In SearchActivity error?

### ISSUE THREAD

**Issue by DmitrySoloviyev** [amitshekhariitbhu/RxJava2-Android-Samples#31]
When I click on keyboard button "actionSearch" observer call "onComplete" and observable do not react on text changed event.

**Comment by Vignesh150493** [comment]
@amitshekhariitbhu Yes. The observer(subject) completes in onQueryTextSubmit, and then subsequent onNext calls dont work. 
Any action performed, after text submission dosent work.
Maybe onComplete should not be called in onQueryTextSubmit?

**Comment by amitshekhariitbhu** [comment]
As the current project is just a sample project. But yes, we can make it better here. There are two ways to do it.

- Do not call onComplete.
- Another solution can be resubscribe on it.
